apps/api/src/app/storage/e2e/get-signed-url.e2e.ts

```diff
@@ -1,7 +1,7 @@
 import { UserSession } from '@novu/testing';
 import { expect } from 'chai';
 
-describe('Get Signed Url - /storage/upload-url (GET) @skip-in-ee', function () {
+describe('Get Signed Url - /storage/upload-url (GET)', function () {
```

Comment:
I dont think that this is a problem in EE enabled, the functionality was missing with community edition no?

Reply:
Correct.